docs/workflow-controller-configmap.yaml

```diff
@@ -168,6 +168,25 @@ data:
           name: argo-postgres-config
           key: password
 
+    # Single sign-on
+    sso :
+      issuer: http://dex:5556/dex
+      clientId: argo-server
+      clientSecret:
+        name: argo-server-sso
+        key: clientSecret
+      redirectUrl: https://localhost:2746/oauth2/callback
+
+    # With SSO, RBAC configuration is mandatory
+    rbac:
+      # For each rule...
+      rules:
+        # If the user is in one of these groups.
+        - groups:
+          - authors
+          # Then use this service account.
+          serviceAccount: argo-server
```

Comment:
How about `serviceAccount: <namespace>:<sa>`? `namespace:` is optional.

Reply:
I did ponder this. IT should probably be a `LocalObjectReference`, but I think we should mandate service account set-up in the `argo` namespace because only the operator should set it up.